src/modules/50-assessments/components/TopNav/TopNav.tsx

```diff
@@ -0,0 +1,40 @@
+import { Container, Text } from '@harness/uicore'
+import { Color } from '@harness/design-system'
+import React, { useCallback } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useStrings } from 'framework/strings'
+import harnessLogo from '../../assets/HarnessLogo.svg'
+import harnessTextLogo from '../../assets/HarnessTextLogo.svg'
+import css from './TopNav.module.scss'
+
+const TopNav = (): JSX.Element => {
+  const history = useHistory()
+  const { getString } = useStrings()
+
+  const handleClick = useCallback(() => {
+    history.push('/assessment')
```

Comment:
```suggestion
    history.push('/assessment')
```
history.push should generally take a function which returns the path routes.ToSomeroute. Same issue with other places too in this PR